features/importing/work_import.feature

```diff
@@ -220,3 +220,23 @@ Feature: Import Works
     When I import "http://www.intimations.org/fanfic/idol/Huddling.html"
     Then I should see "Preview"
       And I should see "English"
+
+  Scenario: Searching for a work url should be cached
+    Then there should be no work_url cached for "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"
```

Comment:
Personally, I would rename the step to `there is no cached work_url for "<url>"`. Then the test can go:

```
Given there is no cached work_url for ...
  When I import "<url>"
  And I <post without preview>
Then there is no cached work_url for ...
When I look for a work with url ...
Then there is a cached work_url for...
```

(Ideally, we'd have perfect grammar with both a "there is " and "there should be " but this was a compromise that might avoid people misunderstanding the purpose of the test)

Not a deal breaker though - I agree that the URL search is painfully slow so looking forward to having this!